fsw/public_inc/crypto.h

```diff
@@ -40,7 +40,8 @@ ivv-itc@lists.nasa.gov
 // Initialization
 extern int32 Crypto_Init(void);
 // Telecommand (TC)
-extern int32 Crypto_TC_ApplySecurity(char** ingest, int* len_ingest);
+extern int32 Crypto_TC_ApplySecurity(const uint8* in_frame, const uint32 in_frame_length, \
+                                      uint8 **enc_frame, uint32 *enc_frame_len);
```

Comment:
Just something to think about, not really going to stop the review because of it... Do we really need a 32 bit int field for a TC frame? A frames max size is 1024 bytes so 16 is more than enough.

Reply:
Good thought, it is unnecessary. Resolved in upcoming merge request 
[cutright_dev e422f85]